server/src/entity/Restaurant.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  OneToMany,
+  JoinTable,
+} from 'typeorm';
+import { Menu } from './Menu';
+
+export enum cuisineTypes {
+  AMERICAN = 'American',
+  CHINESE = 'Chinese',
+  JAPANESE = 'Japanese',
+  MEXICAN = 'Mexican',
+  //SEAFOOD = "Seafood",
+  //CAJUN = "Cajun",
+  //KOREAN = "Korean",
+  //MEDITERRANEAN = "Meditterranean",
+  //CUBAN = "Cuban",
+  //MIDDLE_EASTERN = "Middle Eastern",
+  //SUSHI_BAR = "Sushi Bar",
+  //AMERICAN = "American",
+  //FRENCH = "French",
+  //DELIS = "Delis",
+  //SANDWICH = "Sandwich",
+  //BAGEL = "Bagel",
+  //STEAKHOUSE = "Steakhouse",
+  //THAI = "Thai",
+  //UNKNOWN = "Unknown"
+}
+
+@Entity()
+export class Restaurant extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @Column({
+    length: 50,
+  })
+  public name: string;
+
+  @Column({
+    unique: true,
+    nullable: true,
+  })
+  public email: string;
+
+  @Column({
+    unique: true,
+  })
+  public location: string;
+
+  @Column({
+    default: null,
+  })
+  public imgPath: string;
+
+  @Column({
+    default: null,
+  })
+  public phoneNum: string;
+
+  @Column({
+    type: 'float',
+    default: 0,
+  })
+  public review: number;
+
+  @Column({
+    type: 'enum',
+    enum: cuisineTypes,
+    default: cuisineTypes.AMERICAN,
+  })
+  public cuisineType: cuisineTypes;
+
+  @Column({
+    type: 'simple-json',
+    default: {
+      Monday: '07:00 - 21:00',
```

Comment:
You should have them as subkeys i.e.
```
Monday: {
   startTime: 07:00
   endTime: 21:00
}
```
This will allow us to do other things such as see when the restaurant ends without having the parse the string. Also, look into supporting javascript time objects.